package.json

```diff
@@ -4,8 +4,9 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "dev": "nodemon ./bin/www & npm run scss",
-    "scss": "sass --watch src/scss/main.scss public/stylesheets/style.css"
+    "dev": "nodemon ./bin/www & npm run scss & npm run js",
+    "scss": "sass --watch src/scss/main.scss public/stylesheets/govuk-frontend.min.css --style=compressed",
+    "js": "cp -R node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js* public/javascripts"
```

Comment:
May be we can have a compile script:
`"compile": "sass src/scss/main.scss public/stylesheets/style.css && cp -R node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js* public/javascripts"`

Reply:
OHHH I like this, adding that one in.

Reply:
Actually I tried that and it's massive! I think separating and isolating them is fine and allows use to catch any script that fails.

Maintenance wise this also makes sense than having one long script.